app/Main.hs

```diff
@@ -4,25 +4,10 @@ module Main where
 import Web.Spock
 import Web.Spock.Config
 
-import Control.Monad.Trans
-import Data.Monoid
-import Data.IORef
-import qualified Data.Text as T
-
-data MySession = EmptySession
-data MyAppState = DummyAppState (IORef Int)
-
 main :: IO ()
-main =
-    do ref <- newIORef 0
-       spockCfg <- defaultSpockCfg EmptySession PCNoDatabase (DummyAppState ref)
-       runSpock 8080 (spock spockCfg app)
+main = do
+  spockCfg <- defaultSpockCfg () PCNoDatabase ()
+  runSpock 8080 (spock spockCfg app)
```

Comment:
I will extract it later to an environment variable. It's a bit more work than worth it in this PR.